UHResidentInformationAPI/V1/Domain/ResidentInformation.cs

```diff
@@ -21,10 +21,8 @@ public class ResidentInformation
 
     public class Address
     {
+        public string PropertyRef { get; set; }
         public string AddressLine1 { get; set; }
-        public string AddressLine2 { get; set; }
```

Comment:
Removed these because we don't have multiple address columns in the property table.